core/jobs.py

```diff
@@ -577,6 +659,8 @@ def _entity_created_before_job_queued(entity):
         Mapper methods may want to use this as a precomputation check,
         especially if the datastore classes being iterated over are append-only
         event logs.
+        Returns:
+            Bool. If job_queued_msec >= created_on_msec.
```

Comment:
`bool. Whether ....`

What are job_queued_msec and created_on_msec? I don't see them in the arg list, so try to explain this in human-readable terms.